### PR TITLE
improves the translation system

### PR DESCRIPTION
**Pull request by mathdatech**
Improve the translation system
Add a language form in the configure page

### CODE REVIEW

**Comment by kalvn** [comment]
Thanks a lot for this PR!

File: material/linklist.html [hunk 1]
```diff
@@ -111,10 +111,10 @@
                             {if="isLoggedIn()"}
                             <form method="GET"><input type="hidden" name="lf_linkdate" value="{$value.id}">
                                 <input type="hidden" name="token" value="{$token}"><input type="hidden" name="delete_link">
-                                <button type="submit" alt="Delete" title="Delete" class="icon-delete button-delete"></button>
+                                <button type="submit" alt="Delete" title="t('Delete')" class="icon-delete button-delete"></button>
```
Comment (kalvn) [inline material/linklist.html]: To be changed to {'Remove tag'|t}

File: material/linklist.html [hunk 2]
```diff
@@ -111,10 +111,10 @@
...
                             </form>
                             <form method="GET"><input type="hidden" name="edit_link" value="{$value.id}">
-                                <button type="submit" alt="Edit" title="Edit" class="icon-edit"></button>
+                                <button type="submit" alt="Edit" title="t('Edit')" class="icon-edit"></button>
```
Comment (kalvn) [inline material/linklist.html]: To be changed to {'Edit'|t}

**Comment by kalvn** [comment]
@mathdatech any chance that you can fix the 2 tiny typos? Then I'll be able to merge your PR.

**Comment by mathdatech** [comment]
ok done !
sorry, i'm not a git master, just a beginner ;-)

**Comment by kalvn** [comment]
No problem :)
Thanks again for your work!

**Comment by kalvn** [comment]
Unfortunately, it looks like your last commit didn't contain any change. You maybe forgot to add files before the commit.

File: material/linklist.html (hunk 1 above)
Comment (mathdatech) [reply material/linklist.html]: done

File: material/linklist.html (hunk 2 above)
Comment (mathdatech) [reply material/linklist.html]: done

**Comment by mathdatech** [comment]
OK, I understood my problem. Now, it will be done

**Comment by kalvn** [comment]
Argh, still not good, you created `linklist.html` instead of editing `material/linklist.html`.

I suggest that you have a look at [this link](https://stackoverflow.com/questions/1338728/delete-commits-from-a-branch-in-git) to remove the last 3 commits (only keep the first one). 

It should give something like:

```bash
$ git reset --hard 987ac574b4a667773b1ce42afc1f285cf62acee6
$ git push origin HEAD --force
```

Then you edit the file `material/linklist.html` in order to fix the 2 wrong labels and then you redo a commit.

**Comment by mathdatech** [comment]
thank you for your patience, here is the right version
It was a good exercise to learn how to use github.

**Comment by kalvn** [comment]
Perfect! No problem, GIT is not an easy thing to handle ;)

**Pull request merged**